CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 # GoodData Ruby SDK Changelog
+## 1.1.0
```

Comment:
This is not the whole changelog for 1.1.0. Try removing the automerged git flow release tags (158, 159...) and try again